src/docs/content/builders/pin-input.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Pin Input
+title: PIN Input
```

Comment:
Pin Input is fine IMO

Reply:
"Pin" is a word in English with a separate meaning. The correct way to write this is PIN.

Reply:
If we changed it to PIN, we'd also have to rename the method to `createPINInput`, which is quite hard to read. 

Even if Pin is not the most correct form, it's easily understandable in context. We can keep it as it was before.

Reply:
You don't have to rename the method. A common convention with acronyms in camel case is to capitalize only the first letter of the acronym.

Reply:
No qualms there! PIN Input it is. Just make sure the sidebar also reads PIN Input 😄 

Also, maybe it'd make sense to rename this to OTP Input in the future. Thoughts @huntabyte ?

Reply:
I think PIN is more common & recognizable atm!